proxy/src/auth/backend.rs

```diff
@@ -126,16 +123,24 @@ pub struct ComputeCredentials<T> {
     pub keys: T,
 }
 
+#[derive(Debug, Clone)]
 pub struct ComputeUserInfoNoEndpoint {
     pub user: SmolStr,
-    pub cache_key: SmolStr,
+    pub options: NeonOptions,
 }
 
+#[derive(Debug, Clone)]
 pub struct ComputeUserInfo {
     pub endpoint: SmolStr,
     pub inner: ComputeUserInfoNoEndpoint,
 }
 
+impl ComputeUserInfo {
```

Comment:
Could you also add getter for user_name as well?

Reply:
Hmm, what do you think it would be useful for?

Reply:
It was for the old version, where user_name was inside inner structure.
Now looks good.